litebird_sim/detectors.py

```diff
@@ -74,6 +75,11 @@ class DetectorInfo:
         - bandwidth_ghz (float): The bandwidth of the detector, in
              GHz. The default is 0.0
 
+        - band_freqs_ghz (float): band sampled frequencies, in GHz.
+             The default is None
+
+        - band_weights (float): band profile. The default is None
```

Comment:
The docstring says that these values are `(float)`, but I think they are *arrays* of floats, correct?

Reply:
see  `5141dc060`